src/components/theme/guide/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { Component, h } from "@stencil/core"
+
+@Component({
+	tag: "smoothly-theme-guide",
+	styleUrl: "style.css",
+	scoped: true,
+})
+export class SmoothlyThemeColors {
+	render() {
+		return [
+			<smoothly-display>
+				Tint is used as background to highlight, for example hovering an item in the select menu.
```

Comment:
Probably want: "Tint is used for highlighting, for example.."

Reply:
"Tint is used for highlighting as the background..."?

Reply:
I want to make it clear that it is used as a background.

Reply:
But really i am being a bit nitpicky, we probably want to add more information and or revise with more people anyway